src/amo/components/Addon/index.js

```diff
@@ -408,9 +502,13 @@ export class AddonBase extends React.Component {
 export function mapStateToProps(state, ownProps) {
   const { slug } = ownProps.params;
   const addon = state.addons[slug];
+
   let installedAddon = {};
+  let addonsByAuthors = [];
```

Comment:
I think this should default to `undefined` which will be consistent with when they haven't loaded yet. I.E. `state.addonsByAuthors.byAddonSlug[addon.slug]` will return `undefined` before they are loaded.